chart/postgresql/values.yaml

```diff
@@ -8,6 +8,14 @@ global:
 #   imagePullSecrets:
 #     - myRegistryKeySecretName
 
+  migratorJob:
```

Comment:
Can you separate the chart of the DB which comes form OSS and the cart of the dump job in two different subcharts in the local db chart. Similarly to how it was in the compass chart, two sub-charts, one for postgresql and one for migrator.